### ISSUE TITLE
Wrong keysym is reported

### ISSUE THREAD

**Issue by mad-s** [tomaka/winit#273]
Most keys generate different keysyms depending on modifier state, the most prominent example being the shift key. Winit only does these transformations for text input, but some keyboard layouts like Neo put non-character keys like arrow keys on higher layers. Only the unmodified key gets through and the application is sent a KeyboardInput event with virtual keycodes of L,I,A,E in the case of Neo, but no RecievedCharacter event. Correct behaviour would be to send KeyboardInputs for Up, Left, Down, Right keycodes.

**Comment by chetgurevitch** [comment]
ref https://github.com/jwilm/alacritty/issues/818

#274 broke modifiers for tab. The application receives an unmodified tab and then a seperate keyboardinput with virtual_keycode: None and modifier shift. RecievedCharacter events are identical.

**Comment by mad-s** [comment]
That is probably due to the keyboard layout generating ISO_Left_Tab. You can fix it temporarily (until #308 is merged) by adjusting your keyboard layout, for example in `/usr/share/X11/xkb/symbols/pc` change the line
```
key  <TAB> {	[ Tab,	ISO_Left_Tab	]	};
```
to
```
key  <TAB> {	[ Tab	]	};
```
